src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -254,6 +254,8 @@ public static void registerItems(RegistryEvent.Register<Item> event)
             simpleItems.add(register(r, "wood/boat/" + wood.getRegistryName().getPath(), new ItemBoatTFC(wood), CT_WOOD));
         }
 
+        simpleItems.add(register(r, "wood/stick_bunch", new ItemMisc(Size.NORMAL, Weight.LIGHT), CT_WOOD));
```

Comment:
This should not be under `wood`, it's not a variant based item.

Reply:
Thanks for clarifying.